src/sentry/static/sentry/app/components/sidebar/sidebarItem.tsx

```diff
@@ -104,6 +108,7 @@ const SidebarItem = ({
         className={className}
         onClick={(event: React.MouseEvent<HTMLAnchorElement>) => {
           typeof onClick === 'function' && onClick(id, event);
+          showIsNew && localStorage.setItem(isNewSeenKey, 'true');
```

Comment:
Could store a `1`.

Reply:
Yeah I tried this, but the interface for setItem is `string` only 🤔